lib/peak_tracker_web/controllers/peak_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule PeakTrackerWeb.PeakController do
+  use PeakTrackerWeb, :controller
+
+  alias PeakTracker.Mountains
+
+  def index(conn, _params) do
+    peaks = Mountains.Peak
+    |> Ash.Query.sort([:name])
```

Comment:
[@Doerge](https://github.com/Doerge) is it okay to put an Ash.Query statement in a controller or should this be encapsulated in the the module?